sdk/core/azure-core/src/datetime.cpp

```diff
@@ -92,6 +92,11 @@ constexpr ComputeYearResult ComputeYear(int64_t secondsSince1601)
   int secondsInt = static_cast<int>(secondsLeft - static_cast<int64_t>(year4) * SecondsIn4Years);
 
   int year1 = secondsInt / SecondsInYear;
+  if (year1 == 4)
+  {
+    // this is the last day in a leap year
+    year1 = 3;
```

Comment:
This comment is a little confusing since we are talking about a day, but the variable is year.
https://github.com/microsoft/cpprestsdk/pull/1550#discussion_r550737546

Reply:
Feel free to raise any suggestions :). I don't have a more concise & precise way of describing this if statement.